Description: Fix two errors in lxc-netstat
 s/lxc_name/name/
 --exec does not require an argument
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Forwarded: yes
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/lxc/+bug/1011739

Index: lxc-0.8.0~rc1/src/lxc/lxc-netstat.in
===================================================================
--- lxc-0.8.0~rc1.orig/src/lxc/lxc-netstat.in	2012-06-11 17:34:27.000000000 +0000
+++ lxc-0.8.0~rc1/src/lxc/lxc-netstat.in	2012-06-11 17:50:31.774178554 +0000
@@ -19,7 +19,7 @@
 }
 
 shortoptions='hn:'
-longoptions='help,name:,exec:'
+longoptions='help,name:,exec'
 
 getopt=$(getopt -o $shortoptions --longoptions  $longoptions -- "$@")
 if [ $? != 0 ]; then
@@ -39,7 +39,7 @@
 		;;
 	    -n|--name)
 		shift
-		lxc_name=$1
+		name=$1
 		shift
 		;;
 	    --exec)
